adhocteam/Head-Start-TTADP
# add and document maintenance mode

**Description of change**
Adds script for putting the application into maintenance mode and documentation on how to use the script.

**Note**: this will require you to have cf-cli v7 installed.

**How to test**
1. Open either the dev or sandbox enviroments in your web browser and verify that the app is reachable (ex. https://tta-smarthub-sandbox.app.cloud.gov/)
2. Turn on maintenance mode using the script (ex. `./bin/maintenance -e sandbox -m on`)
3. refresh the page in your browser and make sure that the url is now pointing to the maintenance page
4. ~~make sure the app is still available at the maintenance url returned by the script.~~ we are nixing the maintenance routes
5. Turn off maintenance mode using the script (ex. `./bin/maintenance -e sandbox -m off`)
6. refresh the page in your browser and make sure the application is available at the main url again.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/302

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
